tests/google-cloud-python/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: all
+all:
+	docker build --no-cache .
```

Comment:
Can you add a comment describing why you do --no-cache here but not for the other test directories?